scripts/qa_crowbarsetup.sh

```diff
@@ -1715,6 +1723,10 @@ function custom_configuration()
             fi
         ;;
         database)
+            if [ -n "$want_sles12_controller" ] && iscloudver 6plus ; then
```

Comment:
I think we should change `set_proposalvars` to do this:
* make sure `$want_sles12_controller` is unset if we're not in the `iscloudver 6plus`
* make sure `$want_sles` is set if `$want_sles12_controller` is set